src/Promitor.Core.Scraping/Promitor.Core.Scraping.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Promitor.Agents.Core\Promitor.Agents.Core.csproj" />
```

Comment:
What do you need from agent core here? Typically it would be the other way around.

Reply:
The configuration section, where we have `AuthenticationConfiguration` and `AuthenticationMode`

Reply:
I'll merge it as is and quickly check it out